package.json

```diff
@@ -10,8 +10,8 @@
   "scripts": {
     "start": "node index.js",
     "dev": "NODE_ENV=development nodemon index.js | pino-pretty",
-    "lint": "eslint --ext *.js --ext test/*.js",
-    "fix": "eslint --fix --ext *.js --ext test/*.js",
+    "lint": "eslint",
```

Comment:
In case anyone is also looking over this late and not sure of the context, with eslint 9, no longer specify with flat file config